scripts/docker-compose-mysql/compose/cromwell/bin/test-deadlock.sh

```diff
@@ -20,17 +20,30 @@ for host in "${cromwell_hosts[@]}"; do
 done
 
 # Submit batches of workflows to the load balancer.
-for _ in $(seq 50); do
+BATCHES_TO_RUN=100
+WF_PER_BATCH=10
```

Comment:
TOL it's not immediately clear that the `BATCHES_TO_RUN` variable controls the number of batches to run but `WF_PER_BATCH` only describes the number of empty inputs currently hardcoded into `workflowInputs`.  These could be synced with something like

```shell
inputs=$(for ((i=1; i <= $WF_PER_BATCH; i++)); do echo -n '{},'; done)
.
.
.
-F "workflowInputs=[${inputs%?}]"
```